wot-wg-2023-draft.html

```diff
@@ -193,18 +194,29 @@ <h1 id="title">PROPOSED Web of Things Working Group Charter</h1>
         WoT IG <a href="https://w3c.github.io/wot/current-practices/wot-practices.html#plugfests">PlugFests</a>
         during development.
         Open-source implementations will be prioritized.
+-->
       </p>
     </div>
 
     <section id="scope" class="scope">
       <h2>Scope</h2>
       <!-- What exactly is in scope, and out of scope.
           For legal review. Be brief. -->
-      <p>The Working Group has the following work items in scope:</p>
+      <p>During this charter, the Working Group would:</p>
+      <ul>
+      <li>Update the Architecture, Thing Description, Profile, and Discovery specifications
```

Comment:
binding templates is missing here but it is not an update. 

Also, I still think (as also pointed out in TAG review of Arch spec) that Arch spec should become informative.